main/src/cgeo/geocaching/storage/LocalStorage.java

```diff
@@ -46,6 +46,7 @@
     private static final String GEOCACHE_DATA_DIR_NAME = "GeocacheData";
     private static final String OFFLINE_LOG_IMAGES_DIR_NAME = "OfflineLogImages";
     private static final String MAP_THEME_INTERNAL_DIR_NAME = "MapThemeData";
+    private static final String MAPSFORGE_SVG_CACHE_DIR_NAME = "mapsforge-svg-cache";
     private static final long LOW_DISKSPACE_THRESHOLD = 1024 * 1024 * 100; // 100 MB in bytes
```

Comment:
Sorry for asking to late:
Is this still inside some user-inaccessible area or does this target to our base dir or additional geocache data directory?
IMHO those are cache files and should stay internal (isn't there a specific cache dir assigned to each app by the system).

Reply:
Still inside the internal dir but not in the special android cache dir. Same like all the other dirs in the list.

The idea to move stuff to Android cache dir is worth exploring (there's more candidates than just the mapsforge SVG), but maybe worth its own issue

Reply:
> The idea to move stuff to Android cache dir is worth exploring (there's more candidates than just the mapsforge SVG), but maybe worth its own issue


Yes, that would probably be a good idea!

Even though I have reinstalled cgeo recently, there's already 1.5GB content in the cgeo internal directory again.

Using the caching directory would have the benefit of having an easy way to delete cached files without touching important data like preferences or the DB.

![Screenshot_20211122_073508_com android settings_copy_432x936](https://user-images.githubusercontent.com/64581222/142815757-64bb3606-8f4e-42d4-b134-7d425cf8562f.jpg)

Reply:
I believe the vast majority of the internal space is consumed by downloaded cache data in dir `GeocacheData` (especially images from listings). The cache directory seems to contains just data from Google Maps, leakcanary and some debug stuff.

Reply:
@fm-sys 
AFAIK these figures include the "additional geocache data" located in user readable store but not only the internal data (like theme cache, database, etc.)

Reply:
@Lineflyer I checked on my device. I have around 5000 caches stored offline, and the total space consumed shown is about 1GB. This is around 200KB per cache. c:geo downloads and stores all images of a listing in the GeocacheData dir (together with some meta-info e.g. etag and last-modified), and this data alone should make up the larger part of the 200KB in average. If someone turns on saving of log images, the data size will surely become much more

Why not put this data in the cache dir?

Reply:
> @fm-sys
AFAIK these figures include the "additional geocache data" located in user readable store but not only the internal data (like theme cache, database, etc.)

The external/public directory is more than 3GB on my device cause of quite some offline maps. So I believe this folder isn't included in these figures on my device...

> and this data alone should make up the larger part of the 200KB in average. If someone turns on saving of log images, the data size will surely become much more
>
>Why not put this data in the cache dir?

At least if the setting is enabled, pictures shouldn't be deleted automatically by the system. But in all other cases these files should IMHO go into the app-cache directory...

Reply:
Lets take the topic "Usage of system cache dir" into an own issue. Will open it.